src/bin/js/worker_js.ml

```diff
@@ -259,11 +146,12 @@ let main worker_id content =
       Worker_interface.diagnostic =
         Some [Format.asprintf "%a" Errors.report e]
     }
-  | _ ->
+  | exn ->
     let res = Worker_interface.init_results () in
+    let msg = Fmt.str "Unknown error: %s" (Printexc.to_string exn) in
```

Comment:
:heart:

Reply:
It is still better for the ones we do catch — I'll take an improvement over what we had before even if it is not perfect!